src/components/course/CourseContextProvider.jsx

```diff
@@ -22,6 +22,7 @@ export const CourseContextProvider = ({
   userCanRequestSubsidyForCourse,
   coursePrice,
   currency,
+  canOnlyViewHighlightSets,
```

Comment:
[context, pun intended] Exposing this bool via `CourseContext` so its more easily accessible through subcomponents under the course page routes.